database-bridge-lambda/src/sql/Item.ts

```diff
@@ -186,3 +187,16 @@ export const getItemCounts = (
     WHERE "pinboardId" IN ${sql(args.pinboardIds)} 
     GROUP BY "pinboardId"
 `;
+
+export const getUniqueUsersPerHourInRange = async (
+  sql: Sql,
+  range: Range
+) => sql`
+    SELECT
+        DATE_TRUNC('hour', "timestamp") as "hour",
+        COUNT(DISTINCT "userEmail") as "uniqueUsers"
+    FROM "Item"
+    WHERE
+        "timestamp" >= ${range.from} AND "timestamp" < ${range.to}
+    GROUP BY "hour"
+`;
```

Comment:
```suggestion
`.values();
```
as per https://github.com/porsager/postgres#rows-as-array-of-values it returns an array of arrays here, which should allow you to delete the `mappingService.ts` and its test.

Reply:
Nice. We'll update that 👍